backend/package.json

```diff
@@ -25,6 +25,7 @@
     "helmet": "^3.22.0",
     "node-sass": "^4.14.1",
     "node-sass-json-importer": "^4.3.0",
+    "socket.io": "^3.1.2",
```

Comment:
why did you add this module?

Reply:
These changes were automatically been made while making the docker file

Reply:
@ShubhamPatel33, please remove the socket io dependency as it is not required. And sorry for the slow reviews, have been a little occupied lately.

Reply:
Removed the socket.io,will it gonna count in Mwoc as the PR was made in the mentioned date and the changes took some time .